site/src/theme.ts

```diff
@@ -5,6 +5,13 @@ const createBreakpoint = (value: number) => {
     };
 };
 
+export enum BreakpointValue {
+    sx = 600,
+    sm = 900,
+    md = 1200,
+    lg = 1600,
+}
```

Comment:
I'd prefer an object here 😅 

```suggestion
export const breakpointValue = {
    sx: 600,
    sm: 900,
    md: 1200,
    lg: 1600,
};
```